vm/src/obj/objfunction.rs

```diff
@@ -1,5 +1,9 @@
 use super::objcode::PyCodeRef;
 use super::objdict::PyDictRef;
+#[cfg(feature = "jit")]
+use super::objfloat;
+#[cfg(feature = "jit")]
+use super::objint;
```

Comment:
nit; you could condense this into one `#[cfg jit] use super::{objint, objfloat}`